ckanext/collaborators/templates/package/edit_base.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block content_primary_nav %}
+    {{super()}}
+    {% if h.check_access('dataset_collaborator_create', {'id': pkg_dict.id}) %}
+        {{ h.build_nav_icon('collaborators.read', _('Collaborators'), dataset_id=pkg.name)}}
```

Comment:
Add the little users icon with the following:
```suggestion
        {{ h.build_nav_icon('collaborators.read', _('Collaborators'), dataset_id=pkg.name, icon='users')}}
```